wp-content/themes/csisjti/assets/_scss/pages/resource-library.scss

```diff
@@ -122,12 +122,12 @@
     }
   }
 
-  &__sticky {
+  &__side-panel {
```

Comment:
You can remove these styles if you get rid of the `side-panel` div. If you decide to keep the div, this should not have absolute positioning - we want it to be in the document flow.